test/tools/api-client/python/lib/test_bmapi.py

```diff
@@ -127,7 +127,7 @@ def test_create_game(self):
   def test_load_game_data(self):
     known_keys = [
       'activePlayerIdx', 'currentPlayerIdx', 'description',
-      'gameActionLog', 'gameActionLogCount',
+      'dieBackgroundType', 'gameActionLog', 'gameActionLogCount',
```

Comment:
Nit: this list is supposed to be alphabetized.  I'm not going to block the pull over it, though, it's a test config, whatever.